numpy/ma/core.py

```diff
@@ -28,12 +28,14 @@
 
 import numpy as np
 import numpy.core.umath as umath
+from numpy.core import multiarray as mu
 import numpy.core.numerictypes as ntypes
 from numpy import ndarray, amax, amin, iscomplexobj, bool_
 from numpy import array as narray
 from numpy.lib.function_base import angle
 from numpy.compat import getargspec, formatargspec, long, basestring
 from numpy import expand_dims as n_expand_dims
+from numpy.core._methods import _count_reduce_items
```

Comment:
I don't think we have a convention, but private to file seems most reasonable. I think the idea is that the private functions are implementation details and may change.